docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -143,6 +143,54 @@ This shortcode inserts a lists of links to the page's subpages and includes the
 {{</* section withDescriptions="true"*/>}}
 ```
 
+## `docs/reference` shortcode
+
+The `docs/reference` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
+When content from one repository is published to more than one documentation set, the `docs/reference` shortcode lets you specify appropriate links for each doc set.
+
+For example, alerting documentation is sourced from the Grafana repository and published to both Grafana documentation and Grafana Cloud documentation.
+Any link from alerting documentation to dashboard documentation, for example, which is also published to both Grafana and Grafana Cloud, needs to be specific as to which destination is appropriate for the product documentation.
+Grafana Cloud alerting documentation should link to Grafana Cloud dashboard documentation, and Grafana alerting documentation should link to Grafana dashboard documentation.
+
+The content of the `docs/reference` shortcode must be within opening and closing tags.
+
+| Parameter             | Description                                                                                                                                                                                                                                                               | Required |
```

Comment:
I don't think this table is parallel with the others in the page.

All others are talking specifically about Hugo shortcode parameters. This table is talking about elements of the text within the opening and closing shortcode tags.

Reply:
I think we need a usage example with placeholder text so that readers can understand what `label` or the other rows are referring to.